src/Symfony/Component/Mailer/Transport/AbstractTransport.php

```diff
@@ -31,13 +32,15 @@
 abstract class AbstractTransport implements TransportInterface
 {
     private ?EventDispatcherInterface $dispatcher;
+    protected ?MessageBusInterface $messageBus;
```

Comment:
I suggest making it protected readonly, or to make it private with a protected getter like the logger. Child classes should not need to be able to modify it.